contracts/hts-precompile/README.md

```diff
@@ -0,0 +1,68 @@
+## HTS Precompile methods
+
+The following table highlights the coded HTS precompile methods and their availability in the Hedera network exposed via the `0x167` address.
+
+| Function Name                             | hash          | Consensus Node Release Version | HIP                              | Method Interface | Comments
```

Comment:
For method interface, we are displaying them in two ways, I think we should stick with one for consistency.
Either this:
`allowance(address,address,address)`
or this:
`allowance(address token, address owner, address spender)`

Reply:
Fixed